tools/auto-fuzz/manager.py

```diff
@@ -135,27 +135,37 @@ def project_name(self):
             else:
                 return self.github_url.split("/")[1]
 
-    def write_basefiles(self):
+    def write_basefiles(self, project_build_type=None):
         with open(self.build_script, "w") as bfile:
-            bfile.write(base_files.gen_builder_1(self.language))
+            bfile.write(
+                base_files.gen_builder_1(self.language, project_build_type))
 
         with open(self.base_fuzzer, "w") as ffile:
-            ffile.write(base_files.gen_base_fuzzer(self.language))
+            ffile.write(
```

Comment:
Could we use a better name than `ffile`?

Reply:
Changed..